### ISSUE TITLE
50 epochs usually way too much

### ISSUE THREAD

**Issue by tommiekerssies** [MediaBrain-SJTU/RegAD#23]
Hi, I notice when doing the aggregated training, that usually after 1 or 2 epochs the best img, pixel AUROC is already reached. The many epochs after that only fluctuate around it and do not further improve. Is this something normal?

**Comment by tommiekerssies** [comment]
For example this is the training curves I get for tile with k=2:
![image](https://user-images.githubusercontent.com/6392002/206422452-e8ad4d3a-9c68-4216-b067-7cad5b63a869.png)

**Comment by chaoqinhuang** [comment]
You are correct. In aggregated training, as the aggregated training set is large enough, training several epochs is enough for some categories. In your case, as the detection result has reached >98% AUC after 3 epochs, it is fine to stop the training.

**Comment by tommiekerssies** [comment]
When we change the optimizer, for example to adam, the loss goes to -1 after one or two epochs already. It seems to me the model is collapsing to a trivial solution. By using SGD with the low learning rate, this collapse is maybe just delayed. What are your thoughts on this?

**Comment by tommiekerssies** [comment]
When we change the optimizer, for example to adam, the loss goes to -1 after one or two epochs already. It seems to me the model is collapsing to a trivial solution. By using SGD with the low learning rate, this collapse is maybe just delayed. What are your thoughts on this?

**Comment by tommiekerssies** [comment]
Also, when I include the test object in the training data, performance is actually worse. How could this be?
